package.json

```diff
@@ -6,9 +6,41 @@
     "type": "git",
     "url": "https://github.com/patsplat/javascript-us.git"
   },
-  "version": "1.1.1",
+  "version": "1.2.0",
   "main": "us.js",
+  "scripts": {
+    "test": "tape test/**/*.js | tap-spec"
```

Comment:
Would be better if this ran off the `./node_modules/.bin/tape` and `./node_modules/.bin/tap-spec` so that user doesn't have to install tape globally